base/errorshow.jl

```diff
@@ -59,6 +75,27 @@ function showerror(io::IO, ex::BoundsError)
     Experimental.show_error_hints(io, ex)
 end
 
+Experimental.register_error_hint(BoundsError) do io, ex
+    if ~isdefined(ex, :a)
+        return print(io, "\nNo description of valid indices available.")
```

Comment:
IMO it is not useful to be told that there is no information to be told.

Reply:
My thinking was that this would prompt package authors to extend this function, but you're right, this is useless.